docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -136,17 +148,54 @@ Using the https://mustache.github.io/[Mustache] template syntax `{{variable name
 `alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.actionGroupName`:: The name of the action group of the alert that scheduled the action.
 `alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly.
+`kibanaBaseUrl`:: The configured <<server-publicBaseUrl,`server.publicBaseUrl`>>. If not configured, this will be empty.
+
+[float]
+[[alert-summary-action-variables]]
+===== Action variables for summary of alerts
+
+NOTE: This type of action frequency is not available for all rule types.
+
+When the action frequency is **Summary of alerts**, rules pass the following
+variables:
+
+`rule.id`:: The ID of the rule.
```

Comment:
Do you think it would make sense to have a section for action variables that are available for either summary or individual alert and then having a section for alerts and a section for summary instead of repeating the `rule.*` variables and other common ones?

Reply:
i also thought about it but there are only 6 shared variables and i believe it would just add more complexity.

Reply:
I've tried splitting those shared ones to an earlier part of the section via e6facfb7c296dc543d74a11aa82c6c26e23534e9. If you don't like that, we can revert the change.